WcaOnRails/app/models/format.rb

```diff
@@ -21,8 +21,8 @@ def allowed_first_phase_formats
     {
       "1" => [],
       "2" => ["1"],
-      "3" => ["1", "2"],
-      "m" => ["1", "2"],
+      "3" => ["1"],
```

Comment:
I don't think this line should be changed to 1. In FM and MBF you can allow people to do their first two attempt as a cutoff phase.

Reply:
Yes, good point. Thanks for chiming in!